R/timeseries.R

```diff
@@ -13,28 +13,35 @@
 #' 
 #' @export
 #' @import ggplot2
+#' @importFrom tidyr gather_
 #' 
 #' @examples 
 #' w.use <- wUseSample
 #' year1 <- 2005
 #' year2 <- 2010
 #' years <- c(year1, year2)
 #' areas <- c("Kent County","Sussex County")
-#' data.elements <- "PS-GWPop"
-#' plotObject <- time_series_data(w.use, data.elements, area.column = "COUNTYNAME",areas = areas)
+#' area.column = "COUNTYNAME"
+#' data.elements <- c("PS-GWPop","PS-SWPop")
```

Comment:
These are going to need to change to PS. rather than PS- There will likely be a conflict with the PR I have open right now.